src/foundry/common/prosemirror/highlight-matches-plugin.d.mts

```diff
@@ -74,5 +75,5 @@ declare class ProseMirrorHighlightMatchesPlugin extends ProseMirrorPlugin {
    */
   constructor(schema: Schema, options: ProseMirrorMenu.Options);
 
-  static override build(schema: Schema, options?: Record<string, unknown> | undefined): Plugin;
+  static override build(schema: Schema, options?: EmptyObject | undefined): Plugin;
```

Comment:
This does mean that subclasses can't easily add options. I don't really expect many subclasses though but this requires some judgement.

Reply:
Yeah - I was just matching the other plugins, since it doesn't use options.  For that matter, it doesn't use schema.  

Everything about this one seems strange.  Its constructor also uses `options` from a different plugin (ProseMirrorMenu) and then just assigns them to `this.options`, which as far as I can tell is never used anywhere (or even declared).  But that's all in the source.

That said, it seems unlikely you'd ever need to subclass it.  The entire class is:
```
/**
 * A ProseMirrorPlugin wrapper around the {@link PossibleMatchesTooltip} class.
 * @extends {ProseMirrorPlugin}
 */
export default class ProseMirrorHighlightMatchesPlugin extends ProseMirrorPlugin {
  /**
   * @param {Schema} schema                     The ProseMirror schema.
   * @param {ProseMirrorMenuOptions} [options]  Additional options to configure the plugin's behaviour.
   */
  constructor(schema, options={}) {
    super(schema);
    this.options = options;
  }

  /* -------------------------------------------- */

  /** @inheritdoc */
  static build(schema, options={}) {
    return new Plugin({
      view(editorView) {
        return new PossibleMatchesTooltip(editorView);
      },
      isHighlightMatchesPlugin: true
    });
  }
}
```

So as long as we expose PossibleMatchesTooltip (which in retrospect I'm thinking we should move into the namespace), you could easily recreate the functionality if you needed to.